docusaurus/docs/how-to-guides/ui-extensions/expose-a-component.md

```diff
@@ -11,9 +11,9 @@ keywords:
 sidebar_position: 30
 ---
 
-Expose components to allow app plugins to easily share functionality with other app plugins without extension points. 
+As a content provider, you can expose components from your app plugins to easily share functionality in an extension point.
 
-Compared to [registering an extension](./register-an-extension), when you expose a component you do not require the other extension providers to explicitly register their extensions against any extension points. Therefore the component may be [used by any app plugin](./use-an-exposed-component.md) with no action required by the provider.
+Compared to [registering an extension](./register-an-extension), when you expose a component you do not demand any explicit action to the user rendering the component. Therefore the component can be [used by any extension point](./use-an-exposed-component.md) with no further action required.
```

Comment:
I tried to update the content, but please feel free to update the wording.

```suggestion
Compared to [registering an extension](./register-an-extension), when you expose a component, other plugins can use it wherever they want, instead of you needing to define which extension point you hook it into. This also means that the component needs to be more generic, as it is not targeting a specific extension point.
```